### PR TITLE
build: wishlist-page

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
image

### CODE REVIEW

File: styles/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+.badge-dismiss:hover{
+    color:  var(--ecomm-color-primary);
+}
+.primary-button{   
+    background-color: var(--danger-color);
+    color: var(--button-secondary-color);
+    margin: 5px 5px 0px 5px;
+    width: 50%;
+    font-size: 15px;
```
Comment (Megha-Joshi) [inline styles/wishlist.css:34]: Instead of px use rem

**Review by Megha-Joshi — APPROVED** [review]
Good to merge.

**Review by dev-prabhat — APPROVED** [review]
UI is clean naming of classes are relatable with it's property

File: styles/wishlist.css (hunk 1 above)
Comment (Shubhamkr2610) [reply styles/wishlist.css:34]: I think rem should be used for more than 20px value . I'm not sure  about it, I'll look into it. Thank you for your feedback

**Review by Muzammil-khan710 — APPROVED** [review]
UI is clean and good. Naming is upto the mark.
Good to merge !

**Pull request merged**